serverless/serverless
# Configurable Authorizer Type

## What did you implement:

Closes #4325

I noticed this PR was sitting https://github.com/serverless/serverless/pull/4339. I contacted @realmike33 to see if I could help.



## How did you implement it:

Check for a passed in type value and assign. If empty, default to 'TOKEN'.


## How can we verify it:

Check that an Authorizer was created with type Request instead of Token.

![screen shot 2017-10-10 at 1 47 59 pm](https://user-images.githubusercontent.com/13679/31404442-b40d4e24-adc1-11e7-9178-33b7655b4188.png)



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
